marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -400,8 +402,18 @@ public RunFlowResponse run(String jobId, String[] items, Map<String, Object> opt
                 }
             }
             catch(Exception e) {
-                e.printStackTrace();
-                throw new RuntimeException(e);
+                if(StringUtils.containsIgnoreCase(e.getMessage(), "Plugin error")){
+                    try {
+                        resp = objectMapper.readValue(((MarkLogicServerException)e).getFailedRequest().getMessage(), RunFlowResponse.class);
+                    } catch (IOException ex) {
+                        ex.printStackTrace();
+                        throw new RuntimeException(ex);
+                    }
+                }
+                else{
+                    e.printStackTrace();
```

Comment:
I don't think it's necessary to call printStackTrace in either place here. Since a RuntimeException is being thrown, it will eventually be logged somewhere up the stack.

Reply:
Removed it